client/components/App.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
-import { BrowserRouter as Router, Switch } from 'react-router-dom'
+
+import Login from './Login'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
```

Comment:
Redux changes are not required here. Please remove these changes. They are not part of your ticket.